physics/GFS_GWD_generic.F90

```diff
@@ -29,7 +30,8 @@ subroutine GFS_GWD_generic_pre_run(                               &
       real(kind=kind_phys), intent(in) :: mntvar(im,nmtvr)
 
       real(kind=kind_phys), intent(out) ::                              &
-     &  oc(im), oa4(im,4), clx(im,4),                                   &
+     &  var(im), oc(im), oa4(im,4), clx(im,4),                          &
```

Comment:
Are all these variables allocated all the time, or some only if the drag suite if used? If the latter is the case, these variables must have assumed-size definitions, e.g. something like `foo(:,:)`.

Reply:
When the GSL drag suite is used, var(im), varss(im), ocss(im), oa4ss(im,4) and clxss(im,4) are additional variables.  The variables oc(im), oa4(im), oa4(im,4) and clx(im,4) are used by both the GSL drag suite and the EMC drag suite.

Reply:
The variables that are only allocated when the drag suite is used (see logic in GFS_typedefs.F90) need to be declared as asssumed-size arrays.